.github/workflows/test.yml

```diff
@@ -64,3 +64,21 @@ jobs:
       run: docker build -t bm-no-pi --build-arg IMAGE_TYPE=test -f Dockerfile.noPI .
     - name: Run tests
       run: docker run --rm -w /behavioral-model bm-no-pi /bin/bash -c "make check -j$(nproc)"
+
+  test-ubuntu22:
+    needs: check-changes
+    if: ${{ needs.check-changes.outputs.has_changes == 'yes' }}
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'recursive'
+    - name: Install dependencies
+      run: sudo ./install_deps_ubuntu_22.04.sh
+    - name: Build BMv2
+      run: |
+        ./autogen.sh
+        ./configure --with-pdfixed --with-pi --with-stress-tests --enable-debugger --enable-Werror
+        make -j$(nproc)
+    - name: Run tests
+      run: sudo make check -j$(nproc)
```

Comment:
this is surprising, which tests require superuser privileges?

Reply:
Thank you for pointing this out! I've removed `sudo` as the tests don't require superuser privileges.